builtin/logical/ssh/backend_test.go

```diff
@@ -319,6 +320,118 @@ func TestBackend_allowed_users(t *testing.T) {
 	}
 }
 
+func TestBackend_allowed_users_template(t *testing.T) {
+	testEntityID := "c65c8cff-5142-3d41-93d0-9d06286e7203"
+	testSshUsername := "testsshusername"
+	testEntityMetadata := map[string]string{
+		"ssh_username": testSshUsername,
+	}
+	testAllowedUsersTemplate := "ssh-{{ identity.entity.metadata.ssh_username }}"
+	expectedValidPrincipal := "ssh-" + testSshUsername
+
+	// Setup Vault cluster with a preexisting entity.
+	coreConfig := &vault.CoreConfig{
```

Comment:
Would it be possible to rewrite the setup code in this function to reuse `getSshCaTestCluster` ?